src/styles/App.css

```diff
@@ -37,6 +37,12 @@ div#siaApp {
   display: none;
 }
 
+.issue-title-text {
```

Comment:
The preferred styling approach in Material-UI is `CSS-in-JS`. 

Unless there is a compelling reason to put the style in App.css (e.g. the style is global instead of just a local thing), then we should try to keep the style as close as possible to its target element, and keep the App.css as minimal as possible.

See here for more info: 
https://material-ui-next.com/customization/css-in-js/
https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660
https://medium.freecodecamp.org/css-in-javascript-the-future-of-component-based-styling-70b161a79a32